### DISCUSSION TITLE
Charity Proposal 2023

### DISCUSSION THREAD

**Discussion by CarrotCounter** [sunflower-land/sunflower-land#2157] [General]
Sunflower Land is and always has been a community project. 

In line with of our values, players donate a small fee to charity upon minting a new farm. To date Sunflower Land has raised and donated approximately $40k USD to both charities. An extraordinary number that we are very proud of.

For 2023, we are currently seeking recommendations from the community for charities we should partner with. A charity that our donations can have a big impact on people's lives, and the community feel comfortable with aligning with. 

Also, we are looking for a charity that:

- Accepts crypto donations
- Has a Polygon wallet address that is verifiable
- Preferably has a partner network so we can verify the charity and donations

Please share your thoughts and why we should partner with your recommended charity.

**Update:**
As discussed on the Weekly Dev chat on Discord on 28th February, we are looking at partnering with the Giving Block. They provide a solution that allows crypto payments to charity. By using the link below, you can filter for over 2k+ charities that meet our requirements:  https://thegivingblock.com/donate/

Suggestions have been great so far, but most, if not all, do not meet the requirements of accepting crypto payments in MATIC. Please correct me if you have posted a suggestion and it does meet the requirements, or it is visible on Giving Block website.

**Comment by Aquila199** [comment]
i would like to suggest in our current world i think protecting free speech and digital privacy is a growing issue 

https://www.eff.org/

**Reply by bduino** [reply]
I second this!

**Reply by Aquila199** [reply]
@CarrotCounter you replied to the other suggestions does that mean mine is not worthy shame i do think protecting people online is becoming more and more important

**Reply by CarrotCounter** [reply]
Absolutely not! We are considering all the feedback

**Comment by DonatasDee** [comment]
While charity is fantastic and can still be a focus I would also like for the team to consider giving a % of income that was given to charity before now to actual contributors and builders of the game, mods and so on.

People have been giving their passion for free for ages and even the town hall is gone, credits on NFT's on Opensea are also gone.

Prioritize people that helped build what we have now.

**Reply by Aminbtdds** [reply]
Charity and donations are cool and things to be proud of, if and if only people who contribute their lifetime to project get paid properly.

**Reply by adamhannigan** [reply]
Thanks for your feedback - the game will continue to support contributors through in game mechanisms such as airdrops for designers, contributor cosmetics, event donations etc.

We are always proud that 4 of our contracted engineers started as open source contributors, all contracted artists are from the community, 4 contracted Discord admins/moderators were community mods as well ❤️ The community emphasis is such a core aspect of the project that we keep in mind while building 🤗

**Reply by CarrotCounter** [reply]
Thanks for your comment. 

Further to Adam's comment and for full transparency, Sunflower Land does not hold any of the donations. All donations are paid by the players directly to the charities' digital wallet. We understand your feedback and will continue to support our amazing contributors, but this proposal is specifically for the charities we would like to align with for 2023. 

If you have any suggestions, please send them through :)

**Comment by antonioli2** [comment]
I think it's great to help people, so I think a good way is to incorporate an association with a project dedicated to nature conservation. Conserving nature is the best way to help humanity. I know of one in which, for example, they plant trees to reforest destroyed areas, which are linked to a nft of which anyone who wants to contribute to the project can own. Couldn't be more in line with sunflowerland...sowing, nfts...
If you are interested in the project I can give you more information...

**Reply by CarrotCounter** [reply]
Yes, sounds interesting! Can you send information through?

**Comment by IvanSkazkin** [comment]
Please give money to people who help the project. Draw art or moderate.

**Comment by Alaindrea** [comment]
I want to nominate Mercy corps because they are global and innovative like us. And also like Sunflower Land, they make good videos :)

https://thegivingblock.com/donate/mercy-corps/

**Reply by CarrotCounter** [reply]
Thanks for your reply! Do you have any contacts at Mercy Corps?

I like the idea of having a specific project we can donate to, so we can see the benefit of our donations first-hand.

**Reply by Alaindrea** [reply]
I do not. sorry.

**Comment by Cachorrolokocerbero** [comment]
Cuido de aproximadamente 45 gatinhos que foram abandonado ou vítimas de maus tratos,  a única ajuda recebida é da veterinária e de um vendedor que me passa ração mais barata  , agradeceria se fosse levada em consideração uma doação,

**Comment by On1el** [comment]
https://consciousplanet.org

I would like to offer my support for the Save the Soil Project

I don't know if they have a crypto wallet.
But we can do a very good thing if we support their initiative.
Like rewarding people who plant a tree in real life and tagging it with the hashtag #save the soil #sunflowerland 
As a gift you can give NFT with a boost (+1 tree) which you can get only this way

**Comment by BlazinMazon** [comment]
I think we should work on improving the existing mechanics we have in game over new SFT's and tasks. We need use case for lots of things still and the only way to earn SFL directly is through crops this needs to be addressed. Starting new tasks, I feel would only cloud the intermittent progress we've made for SFL currently.

**Comment by brian-man** [comment]
On charities, for the animal onesies can a % of the proceeds go to endangered species charities?
